### PR TITLE
Fix map loading

### PR DESCRIPTION
**Pull request by forestbbbbbbb**
## About The Pull Request

You aren't supposed to tick templates. CI does it in https://github.com/tgstation/TerraGov-Marine-Corps/blob/e2a5b6a3ec3e105f55124110fc824d71a8df352d/_maps/_basemap.dm#L25-L27

## Changelog

:cl:
fix: fixed maps
/:cl:

### CODE REVIEW

**Comment by ivanmixo** [comment]
Oh. Guh.

**Comment by TiviPlus** [comment]
why did you remove admin z lol
I bet thats why your monimaps was "failing"

**Comment by ivanmixo** [comment]
> why did you remove admin z lol I bet thats why your monimaps was "failing"

It was failing on the first commit where it was just the unit box being unticked

File: tgmc.dme [hunk 1]
```diff
@@ -11,8 +11,6 @@
 // END_PREFERENCES
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
-#include "_maps\map_files\generic\Admin_Level.dmm"
```
Comment (Lumipharon) [inline tgmc.dme:14]: ?

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**